.github/workflows/analysis.yml

```diff
@@ -125,7 +125,7 @@ jobs:
 
       - name: Summarize report
         env:
-          MAX_BUGS: 12
+          MAX_BUGS: 16
```

Comment:
Hmm, it was already accounted for in PR introducing Voodoo support in d2d2154413003d7a2be897d796aa3d26b3d48202. Maybe we get new warnings with new clang static analyser or something?